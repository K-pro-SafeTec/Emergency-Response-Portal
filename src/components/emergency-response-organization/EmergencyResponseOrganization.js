import React from 'react';
import NavMenu from '../shared/NavMenu';
import NavMenuItem from '../shared/NavMenuItem';
import IconShield from '../../icons/shield.svg';
import IconHierarchy from '../../icons/hierarchy.svg';
import IconProfileGroup from '../../icons/profile-group.svg';
import IconFileExcel from '../../icons/file-excel.svg';
import IconCogs from '../../icons/cogs.svg';

export default () => (
  <NavMenu>
    <NavMenuItem name="Beredskapslag" icon={IconHierarchy} to="" />
    <NavMenuItem name="Kompetansestatus" icon={IconShield} to="" />
    <NavMenuItem name="Forenlighetsmatrise" icon={IconFileExcel} to="" />
    <NavMenuItem name="Stedfortredere" icon={IconProfileGroup} to="" />
    <NavMenuItem name="Reserveressurser" icon={IconCogs} to="" />
  </NavMenu>
);
