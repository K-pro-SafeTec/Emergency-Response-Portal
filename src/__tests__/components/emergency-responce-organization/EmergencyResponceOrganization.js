import React from 'react';
import TestRenderer from 'react-test-renderer';
import { MemoryRouter as Router } from 'react-router-dom'
import EmergencyResponseOrganization
  from "../../../components/emergency-response-organization/EmergencyResponseOrganization";

let testRenderer;

beforeAll(() => {
  testRenderer = TestRenderer.create(
    <Router>
      <EmergencyResponseOrganization  match={{}} />
    </Router>
  );
});

test('renders correctly', () => {
  const tree = testRenderer.toJSON();
  expect(tree).toMatchSnapshot();
});
