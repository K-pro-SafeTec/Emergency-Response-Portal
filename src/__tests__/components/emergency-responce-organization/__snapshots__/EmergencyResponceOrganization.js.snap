// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <header
    className="MuiPaper-root-14 MuiPaper-elevation4-20 MuiAppBar-root-5 MuiAppBar-positionFixed-6 MuiAppBar-colorDefault-11 mui-fixed"
  >
    <div
      className="MuiToolbar-root-41 MuiToolbar-regular-43 MuiToolbar-gutters-42"
    >
      <a
        className="MuiButtonBase-root-51 MuiIconButton-root-45 AppPage-homeButton-2"
        href="/"
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex="0"
      >
        <span
          className="MuiIconButton-label-50"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root-54"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M10 20v-6h4v6h5v-8h3L12 3 2 12h3v8z"
            />
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />
          </svg>
        </span>
        <span
          className="MuiTouchRipple-root-127"
        />
      </a>
      <button
        className="MuiButtonBase-root-51 MuiButtonBase-disabled-52 MuiIconButton-root-45 MuiIconButton-disabled-49"
        disabled={true}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex="-1"
        type="button"
      >
        <span
          className="MuiIconButton-label-50"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root-54"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />
            <path
              d="M20 11H7.83l5.59-5.59L12 4l-8 8 8 8 1.41-1.41L7.83 13H20v-2z"
            />
          </svg>
        </span>
      </button>
      <button
        className="MuiButtonBase-root-51 MuiButtonBase-disabled-52 MuiIconButton-root-45 MuiIconButton-disabled-49"
        disabled={true}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex="-1"
        type="button"
      >
        <span
          className="MuiIconButton-label-50"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root-54"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />
            <path
              d="M12 4l-1.41 1.41L16.17 11H4v2h12.17l-5.58 5.59L12 20l8-8z"
            />
          </svg>
        </span>
      </button>
      <h2
        className="MuiTypography-root-63 MuiTypography-title-69 MuiTypography-colorInherit-92 AppPage-grow-1"
      >
        Beredskapsorganisasjon
      </h2>
      <a
        className="AppPage-safetecLink-3"
        href="https://www.safetec.no/"
      >
        <img
          alt="Levert av Safetec"
          height="44px"
          src="safetec.svg"
          width="153px"
        />
      </a>
    </div>
  </header>,
  <div
    className="AppPage-content-4"
  >
    <div
      className="NavMenu-content-98"
    >
      <a
        className="MuiButtonBase-root-51 MuiButton-root-101 MuiButton-outlined-109 MuiButton-sizeLarge-125 NavMenuItem-navigationButton-99"
        href="/emergency-response-teams/"
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        style={
          Object {
            "flexBasis": "30em",
          }
        }
        tabIndex="0"
      >
        <span
          className="MuiButton-label-102"
        >
          <img
            alt="Beredskapslag"
            className="NavMenuItem-icon-100"
            src="response-teams.svg"
          />
          <div>
            Beredskapslag
          </div>
        </span>
        <span
          className="MuiTouchRipple-root-127"
        />
      </a>
      <a
        className="MuiButtonBase-root-51 MuiButton-root-101 MuiButton-outlined-109 MuiButton-sizeLarge-125 NavMenuItem-navigationButton-99"
        href="/compatibility-matrix/"
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        style={
          Object {
            "flexBasis": "30em",
          }
        }
        tabIndex="0"
      >
        <span
          className="MuiButton-label-102"
        >
          <img
            alt="Forenlighetsmatrise"
            className="NavMenuItem-icon-100"
            src="stats-1.svg"
          />
          <div>
            Forenlighetsmatrise
          </div>
        </span>
        <span
          className="MuiTouchRipple-root-127"
        />
      </a>
      <a
        className="MuiButtonBase-root-51 MuiButton-root-101 MuiButton-outlined-109 MuiButton-sizeLarge-125 NavMenuItem-navigationButton-99"
        href="/deputies/"
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        style={
          Object {
            "flexBasis": "30em",
          }
        }
        tabIndex="0"
      >
        <span
          className="MuiButton-label-102"
        >
          <img
            alt="Stedfortredere"
            className="NavMenuItem-icon-100"
            src="deputies.svg"
          />
          <div>
            Stedfortredere
          </div>
        </span>
        <span
          className="MuiTouchRipple-root-127"
        />
      </a>
      <a
        className="MuiButtonBase-root-51 MuiButton-root-101 MuiButton-outlined-109 MuiButton-sizeLarge-125 NavMenuItem-navigationButton-99"
        href="/backup-resources/"
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        style={
          Object {
            "flexBasis": "30em",
          }
        }
        tabIndex="0"
      >
        <span
          className="MuiButton-label-102"
        >
          <img
            alt="Reserveressurser"
            className="NavMenuItem-icon-100"
            src="cogs.svg"
          />
          <div>
            Reserveressurser
          </div>
        </span>
        <span
          className="MuiTouchRipple-root-127"
        />
      </a>
    </div>
  </div>,
]
`;
